cwms-data-api/src/test/resources/cwms/cda/data/sql/users.sql

```diff
@@ -13,6 +13,10 @@ begin
   cwms_sec.add_user_to_group('&webuser','CWMS Users', 'SPK');
   cwms_sec.add_user_to_group('&webuser','CWMS PD Users', 'SPK');
   cwms_sec.add_user_to_group('&webuser','CWMS DBA Users', 'SPK');
+  cwms_sec.add_user_to_group('&webuser','All Users', 'SWT');
```

Comment:
Should this be for all groups? I presumed since there was the SWT user added below that the webuser had permissions.

Reply:
Honestly not sure, but it's generally safe to just call add_user_to_group so it doesn't hurt to just make sure the permissions exist.